files/zh-cn/web/api/element/insertadjacentelement/index.md

```diff
@@ -5,40 +5,38 @@ slug: Web/API/Element/insertAdjacentElement
 
 {{APIRef("DOM")}}
 
-**`insertAdjacentElement()`** 方法将一个给定的元素节点插入到相对于被调用的元素的给定的一个位置。
+**`insertAdjacentElement()`** 方法将一个给定的 {{domxref("Element")}} 元素插入到 DOM 树中的指定位置。
 
 ## 语法
 
 ```
 element.insertAdjacentElement(position, element);
 ```
 
-### 参数
-
 - position
 
-  - : A {{domxref("DOMString")}} 表示相对于该元素的位置；必须是以下字符串之一：
+  - : 一个 {{domxref("DOMString")}}，表示插入内容相对于元素的位置，并且必须是以下字符串之一：
 
-    - `'beforebegin'`: 在该元素本身的前面。
-    - `'afterbegin'`:只在该元素当中，在该元素第一个子孩子前面。
-    - `'beforeend'`:只在该元素当中，在该元素最后一个子孩子后面。
-    - `'afterend'`: 在该元素本身的后面。
+    - `'beforebegin'`：元素自身的前面。
+    - `'afterbegin'`：插入元素内部的第一个子节点之前。
+    - `'beforeend'`：插入元素内部的最后一个子节点之后。
+    - `'afterend'`：元素自身的后面。
 
 - element
-  - : 要插入到树中的元素。
+  - : 要被插入到 DOM 树中的 {{domxref("Element")}} 元素。
 
 ### 返回值
 
-插入的元素，插入失败则返回 null.
+插入的元素，插入失败则返回 `null`。
 
 ### 异常
 
-| 异常          | 说明                           |
-| ------------- | ------------------------------ |
-| `SyntaxError` | 插入的位置是一个无法识别的值。 |
-| `TypeError`   | 插入的元素不是一个有效元素。   |
+- `SyntaxError` {{domxref("DOMException")}}
+  - 无法识别 `position` 的值。
```

Comment:
```suggestion
  - 如果指定的 `position` 不是可识别的值，则抛出该异常。
```

Reply:
Try to fix the DFL in upstream: mdn/content#32317